README.md

```
@@ -103,16 +103,40 @@ python scripts/sandbox_runner.py -- bash -c "cd frontend && pnpm install && pnpm
 
 ## Quick Start (uvx を使用する場合)
 
-リポジトリをクローンせずに、`uvx` を使用して ChronosGraph を MCP サーバーとして即座にセットアップするための最小設定例です。
+リポジトリをクローンせずに、`uvx` を使用して ChronosGraph を MCP サーバーとして即座にセットアップするための最小設定例です。本リポジトリは [release-please](https://github.com/googleapis/release-please) によってタグ付きリリースが作成され、GitHub の tarball アーカイブから直接インストールできます。
 
-#### Claude Desktop 設定例
+#### Claude Desktop 設定例（リリース tarball 版）
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)